packages/radio/src/RadioEditor.tsx

```diff
@@ -75,7 +75,11 @@ export function RadioEditor(props: RadioEditorProps) {
                     {item.label}
                   </Radio>
                   {checked && (
-                    <TextLink as="button" className={styles.clearBtn} onClick={clearOption}>
+                    <TextLink
+                      as="button"
+                      className={styles.clearBtn}
+                      onClick={clearOption}
+                      isDisabled={disabled}>
```

Comment:
Nit: I'd prefer to conditionally render the `clear` button here instead e.g. `!disabled && <TextLink .../>` to reduce visual noise